src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -227,6 +229,16 @@ def setUp(self):
         if self.nvme_parameter == "nvme":
             server_utils.storage_prepare(self.hostlist_servers)
 
+            # Clean daos mnt
+            self.assertTrue(dmg_utils.clean_daos_mnt(self.hostlist_servers))
+
+            # Format hostlist for dmg tool
+            port = self.params.get("port", "/run/server_config/*")
+            servers_with_ports = [
```

Comment:
For now we need to add those port number but later we should use it from server yaml file.I do not see any tests yaml has been updated as part of this PR. I suggest you can update the ior_small.yaml test other vice it's going to fail as no port is updated in ior_small.yaml file.